mapviz_plugins/src/image_plugin.cpp

```diff
@@ -659,6 +703,20 @@ namespace mapviz_plugins
     return units_string;
   }
 
+  std::string ImagePlugin::RotationToString(int rotation)
+  {
+    std::string rotation_string = "0°";
```

Comment:
I'd probably remove the degree characters since they are difficult to type manually.

Reply:
I made the degree symbol optional